uriget/example_test.go

```diff
@@ -51,3 +51,14 @@ func ExampleWithHttpClient() {
 	fmt.Println(err)
 	// Output: failed to make get request: Get "https://example.com": no proxy
 }
+func ExampleGetFile_oci() {
+	testUrl := "oci://ghcr.io/score-spec/score-compose:0.18.0"
```

Comment:
Can we add another couple of test cases:
* no tag
* image digest in place of image tag?

Reply:
why not I'll add more test cases

Reply:
Thank you!